undofx/src/main/java/org/fxmisc/undo/UndoManagerFactory.java

```diff
@@ -14,118 +15,249 @@
 
 public interface UndoManagerFactory {
 
-    <C> UndoManager create(
+    /**
+     * Creates an {@link UndoManager} where changes will never be merged and every change emitted from the
+     * changeStream is considered to be a non-identity change.
+     *
+     * @param changeStream the {@link EventStream} that emits changes
+     * @param invert inverts the change
+     * @param apply applies the change (undo/redo)
```

Comment:
Can you document that then? Otherwise, I feel like this PR will take much longer because I'll write something down and then you'll point out something that needs to be changed.